notes/part01/lab01B.md

```diff
@@ -1,3 +1,17 @@
 # Lab 1B
 
-Populate this content soon ...
\ No newline at end of file
+In this lab, TAs will be working with students to grade labs individually in breakout rooms.
+
+## Lab Grading
+
+There will be a message in Slack that students can reply to in order to sign up for the grading queue. Ensure that your Zoom name matches the name you used to reply on Slack.
+
+Students should have the following three things ready **BEFORE** a TA starts grading:
+
+- Code ready to show
+- Website/assignment ready to demo
+- Terminal/Command prompt/Bash ready to show your branch + latest commits (`git log`)
+
```

Comment:
I'd add something here like:

"We're hoping to spend approximately 10 minutes grading. This will be broken into:
- 5 minutes of demo: you'll show us your code and functioning website (with all the required functionality), answering questions as you go, and taking direction from course staff.
- 5 minutes to answer questions related to the workshop material, which will demonstrate knowledge/understanding.

You'll be able to look at the rubric on the assignment to see how things are broken down."